eng/pipelines/pr.yml

```
@@ -46,3 +46,27 @@ stages:
       scope: lite
     ${{ else }}:
       scope: full
+
+- stage: ValidateSubmodules
+  displayName: 'Validate VMR Submodules'
```

nit: quotation not necessary in YAML (for most cases)

```suggestion
  displayName: Validate VMR Submodules
```